src/unix/bsd/apple/mod.rs

```diff
@@ -1969,6 +1973,18 @@ f! {
     pub fn WIFSTOPPED(status: ::c_int) -> bool {
         _WSTATUS(status) == _WSTOPPED && WSTOPSIG(status) != 0x13
     }
+
+    pub fn EV_SET(ident: ::uintptr_t, filter: kevent_filter, flags: kevent_flags,
```

Comment:
These macros aren't like their C counterparts where they take a `kevent` as the first argument?

Reply:
Oh, yes, I wrote it this way so that you don't need to first create an uninitialized `kevent` struct. But I don't have a strong opinion on this...

Reply:
I think we should stick to the C macros as much as possible